src/components/inline/InputText/InputText.stories.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+
+import InputText from "./InputText";
+
+/*
+For responsive behavior, the width of the parent element
+of this component should be set in a class and should
+follow this example:
```

Comment:
Please follow the typical style for multiline comments in languages that use `/* */` for multiline comments, where each subsequent line should have a `*` that is lined up with the initial `*`. See https://github.com/airbnb/javascript/tree/f0df3a8680479ff0b897cd98a1eab6b156899214#comments for examples.

Also, should this just go on the actual component itself, not the Storybook story?

Reply:
ok